### PR TITLE
[Doc Improvement][Update graph samples][2817941]

### PR DESCRIPTION
**Pull request by v-ypalikila**
Added samples to respective articles as per git issue #13565 .

### CODE REVIEW

File: api-reference/beta/api/chat-post-pinnedmessages.md [hunk 1]
```diff
@@ -114,3 +114,9 @@ Content-Type: application/json
   "id": "1616964509832"
 }
 ```
+
+## Code sample
```
Comment (Lauragra) [inline api-reference/beta/api/chat-post-pinnedmessages.md:118]: @v-ypalikila , we don't add non-standard H2 sections to the API reference topics. Per our CDK guidelines, we follow a standard topic structure to make the content predictable for readers. Please remove these non-standard H2 sections and instead add the links to the samples in the ## See also section (or add the See also section if it's not included - that section is optional). Thanks.

Comment (v-ypalikila) [reply api-reference/beta/api/chat-post-pinnedmessages.md]: Hi @Lauragra,

I've updated the code samples. Removed the heading ## Code sample and moved the links to the see also section.

File: api-reference/beta/api/chat-post-pinnedmessages.md [hunk 2]
```diff
@@ -114,3 +114,8 @@ Content-Type: application/json
   "id": "1616964509832"
 }
 ```
+
+## See also
+
+- [Graph Pinned Message C# sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/graph-pinned-messages/csharp)
```
Comment (Lauragra) [inline api-reference/beta/api/chat-post-pinnedmessages.md:120]: Note that per branding, we always refer to "Microsoft Graph" and never "Graph". For details, please see this Wiki topic: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/398264/Microsoft-Graph-branding-guidelines-and-terminology

Please make sure that the branding is fixed in the corresponding code samples as well.

Also note that Microsoft style is to use sentence case for titles; please follow that with the link text. So for example:

Microsoft Graph pinned message C# sample

Comment (v-ypalikila) [reply api-reference/beta/api/chat-post-pinnedmessages.md]: Hi @Lauragra,

I've updated the branding from Graph to Microsoft Graph and also updated sentence case across all the updated articles.

Thanks,
Yash

File: api-reference/v1.0/resources/teamsapp.md [hunk 3]
```diff
@@ -72,6 +72,8 @@ Users can see these apps in the Microsoft Teams Store, and these apps can be ins
 - [teamsTab](../resources/teamstab.md)
 - [App catalog sample (C#)](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-appcatalog-lifecycle/csharp)
 - [App catalog sample (Node.JS)](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-appcatalog-lifecycle/nodejs)
+- [Microsoft Graph API Teams app catalog life cycle C# sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/graph-appcatalog-lifecycle/csharp)
```
Comment (Lauragra) [inline api-reference/v1.0/resources/teamsapp.md:75]: Global: Please shorten these titles and make lifecycle one word:

Teams app catalog lifecycle C# sample

And so on.

File: concepts/webhooks-with-resource-data.md [hunk 4]
```diff
@@ -601,6 +605,12 @@ let decryptedPayload = decipher.update(base64encodedPayload, 'base64', 'utf8');
 decryptedPayload += decipher.final('utf8');
 ```
 
+## Code sample
```
Comment (Lauragra) [inline concepts/webhooks-with-resource-data.md:608]: Please remove this section and table and add links to the code samples in the See also section.

Comment (v-ypalikila) [reply concepts/webhooks-with-resource-data.md]: removed the section and added to the see also section.

File: api-reference/v1.0/resources/teamsapp.md (hunk 3 above)
Comment (v-ypalikila) [reply api-reference/v1.0/resources/teamsapp.md]: Shortened the titles and updated life cycle as lifecycle.

**Comment by lramosvea** [comment]
Fixes https://github.com/microsoftgraph/microsoft-graph-docs/issues/13565

**Review by Lauragra — APPROVED** [review]

**Pull request merged**